doc/source/ray-core/scheduling/ray-oom-prevention.rst

```diff
@@ -16,117 +16,80 @@ What is the memory monitor?
+The memory monitor is enabled by default and can be disabled by setting the environment variable ``RAY_memory_monitor_refresh_ms`` to zero when Ray starts. 
 
 How do I configure the memory monitor?
 --------------------------------------
 
 The memory monitor is controlled by the following environment variables:
 
-- ``RAY_memory_monitor_refresh_ms (int, defaults to 250)`` is the interval to check memory usage and kill tasks or actors if needed. It is disabled when this value is 0.
+- ``RAY_memory_monitor_refresh_ms (int, defaults to 250)`` is the interval to check memory usage and kill tasks or actors if needed. Task killing is disabled when this value is 0.
 
 - ``RAY_memory_usage_threshold (float, defaults to 0.95)`` is the threshold when the node is beyond the memory
-  capacity. If the memory usage is above this value and the free space is
-  below min_memory_free_bytes then it will start killing processes to free up space. Ranges from [0, 1].
+  capacity. If the memory usage is above this fraction it will start killing processes to free up memory. Ranges from [0, 1].
 
-- ``RAY_task_oom_retries (int, defaults to 15):`` The number of retries for the task or actor when
-  it fails due to the process being killed by the memory monitor.
-  If the task or actor is not retriable then this value is not used. This value is used
-  only when the process is killed by the memory monitor, and the retry counter of the
-  task or actor (:ref:`max_retries <task-fault-tolerance>` or :ref:`max_restarts <actor-fault-tolerance>`) is used when it fails 
-  in other ways. If the process is killed by the operating system OOM killer it will use the task retry and not ``task_oom_retries``.
-  It will retry indefinitely if this value is set to -1.
+Using the Memory Monitor
+------------------------
 
-  When the task is retried due to OOM it applies a delay before re-executing the task. The delay is calculated as
+Retry policy
+~~~~~~~~~~~~
 
-  .. code-block:: bash
+When a task or actor is killed by the memory monitor it will be retried with exponential backoff. The retry delay is calculated as
 
-      delay_seconds = 2 ^ attempt
+.. code-block:: bash
 
-  Where the first retry will be delayed by 1 second as ``attempt`` starts from 0.
+    delay_seconds = 2 ^ (retry_attempt - 1)
 
-  .. note::
+The first ``retry_attempt`` will be delayed by 1 second. There is a cap on the maximum delay, which is 60 seconds.
 
-      Keep the value of ``RAY_task_oom_retries`` low, below 25, to avoid extremely long delays as it is using exponential backoff.
-      The delay is capped at 1 minute.
-  
+This retry is applied only when the worker is killed by the memory monitor. Otherwise, when the worker dies, it uses :ref:`max_retries <task-fault-tolerance>`, or :ref:`max_restarts <actor-fault-tolerance>` and :ref:`max_task_retries <actor-fault-tolerance>` depending on whether it is a task or actor.
```

Comment:
How many times is it retried when things are OOM killed? (infinite)? Can you mention that?